src/containers/NFT/index.js

```diff
@@ -0,0 +1,114 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { withTranslation } from 'react-i18next';
+import { connect } from 'react-redux';
+import NoMatch from '../NoMatch';
+import NFTHeader from './NFTHeader';
+import './styles.css';
+import { analytics, ANALYTIC_TYPES, NOT_FOUND, BAD_REQUEST } from '../shared/utils';
+
+const ERROR_MESSAGES = {};
+ERROR_MESSAGES[NOT_FOUND] = {
+  title: 'assets.no_nfts_message',
+  hints: ['check_nft_id'],
+};
+ERROR_MESSAGES[BAD_REQUEST] = {
+  title: 'invalid_xrpl_address',
+  hints: ['check_nft_id'],
+};
+ERROR_MESSAGES.default = {
+  title: 'generic_error',
+  hints: ['not_your_fault'],
+};
+
+const getErrorMessage = error => ERROR_MESSAGES[error] || ERROR_MESSAGES.default;
```

Comment:
you should use the new `??` operator here instead of `||`. The latter will treat `false` and `''` as false, but the former will only coalesce the specific values `null` and `undefined`

Reply:
thanks that makes sense! done